src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs

```diff
@@ -402,7 +402,8 @@ internal async Task InvokeJobFunction(DateTimeOffset invocationTime, bool isPast
                 if (ScheduleMonitor != null)
                 {
                     await ScheduleMonitor.UpdateStatusAsync(_timerLookupName, ScheduleStatus);
-                    _logger.LogDebug($"Function '{_functionLogName}' updated status: Last='{ScheduleStatus.Last.ToString("o")}', Next='{ScheduleStatus.Next.ToString("o")}', LastUpdated='{ScheduleStatus.LastUpdated.ToString("o")}'");
+                    _logger.LogDebug("Function '{functionName}' updated status: Last='{lastScheduled:o}', Next='{nextScheduled:o}', LastUpdated='{lastUpdated:p}'",
```

Comment:
Possible typo? You have `lastUpdated:p` , I think it should be `:o` (o, not p)

```suggestion
                    _logger.LogDebug("Function '{functionName}' updated status: Last='{lastScheduled:o}', Next='{nextScheduled:o}', LastUpdated='{lastUpdated:o}'",
```

Reply:
🤦

I'm glad you're paying attention!